recipes/2dfatmic/LICENSE

```diff
@@ -0,0 +1,15 @@
+Taken from EPA/600/R-97/052
+August 1997
+Avaialble at https://nepis.epa.gov/Adobe/PDF/P1006YEP.pdf
+
+The U. S. Environmental Protection Agency through its Office of Research and Development
+partially funded and collaborated in the research described here under assistance agreement number
+CR-818322 to The Pennsylvania State University. It has been subjected to the Agency’s peer and
+administrative review and has been approved for publication as an EPA document. Mention of trade
+names or commercial products does not constitute endorsement or recommendation for use.
+
+When available, the software described in this document is supplied on “as-is” basis without
```

Comment:
I don't know if this means this is redistributable.

Reply:
This is a work of the US government and is public domain